src/Aspire.Hosting.Azure/Provisioning/Provisioners/AzureProvisioner.cs

```diff
@@ -266,13 +266,25 @@ private async Task ProcessResourceAsync(IConfiguration configuration, Lazy<Task<
 
     private async Task<ProvisioningContext> GetProvisioningContextAsync(string? userSecretsPath, CancellationToken cancellationToken)
```

Comment:
@oising this fixes https://github.com/dotnet/aspire/issues/1329, which is currently assigned to me. Do you want me to take this change separately?

Reply:
Would you prefer I create a different PR for that? I don't mind doing the work.

Reply:
Yes please can you split it out?

Reply:
@davidfowl  https://github.com/dotnet/aspire/pull/3083